guardian/utils.py

```diff
@@ -133,15 +133,33 @@ def get_40x_or_None(request, perms, obj=None, login_url=None,
                                      redirect_field_name)
 
 
+from django.apps import apps as django_apps
+from django.core.exceptions import ImproperlyConfigured
+
+def get_obj_perm_model_by_conf(setting_name):
+    """
+    Return the model that matches the guardian settings.
+    """
+    try:
+        setting_value = getattr(guardian_settings, setting_name)
+        return django_apps.get_model(setting_value, require_ready=False)
+    except ValueError:
+        raise ImproperlyConfigured("{} must be of the form 'app_label.model_name'".format(setting_value))
+    except LookupError:
+        raise ImproperlyConfigured(
+            "{} refers to model '{}' that has not been installed".format(setting_name, setting_value)
+        )
```

Comment:
It would be nice to use [explicit exception chaining](https://www.python.org/dev/peps/pep-3134/#explicit-exception-chaining).

Reply:
Added explicit exception chaining in 803a26cd87d3767be835947dc11f08406052a0c7

I can't think of any issues with the chaining the exception which is different than [django.contrib.auth.get_user_model()](https://github.com/django/django/blob/abc51d44afcb8482c64c0bb8c1022f11e7eb2bbb/django/contrib/auth/__init__.py#L151-L162)  (but that might just be left over from prior Python2 compatibility)